Source/Extensions/Blazorise.Charts/wwwroot/blazorise.charts.js

```diff
@@ -58,6 +58,9 @@ window.blazoriseCharts = {
 
         if (chart) {
             chart.options = options;
+            chart.aspectRatio = options.aspectRatio ?? chart.aspectRatio;
+            chart.update();
```

Comment:
I would leave only `chart.aspectRatio = options.aspectRatio ?? chart.aspectRatio;` here and then remove update and resize, and make them as a separate call to chart. Update is already available as part of chart API, meaning we need to add `Resize()` method

Reply:
PS. Adding a new API can only be done in 0.9.4. For the support branch, I usually don't allow API changes.